build.sbt

```diff
@@ -527,35 +527,35 @@ lazy val spark = (project in file("connectors/spark"))
     Compile / compile / javacOptions ++= javacRelease11,
     libraryDependencies ++= Seq(
       "org.apache.spark" %% "spark-sql" % sparkVersion % Provided,
-      "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.0",
-      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.0",
-      "com.fasterxml.jackson.core" % "jackson-annotations" % "2.15.0",
-      "com.fasterxml.jackson.core" % "jackson-core" % "2.15.0",
-      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-xml" % "2.15.0",
-      "org.antlr" % "antlr4-runtime" % "4.9.3",
-      "org.antlr" % "antlr4" % "4.9.3",
-      "com.google.cloud.bigdataoss" % "util-hadoop" % "3.0.2" % Provided,
-      "org.apache.hadoop" % "hadoop-azure" % "3.4.0" % Provided,
+      "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
```

Comment:
I'm not sure if things work at runtime if we do this, we had issues with jackson version previously and had to shade the server jar to help with the test cases. see here: https://github.com/unitycatalog/unitycatalog/pull/687/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91L487-L491

Reply:
Do you remember which problem did you run into? Having a single version of Jackson instead of two versions (like it was before)?